xl/playlist.py

```diff
@@ -2115,6 +2115,11 @@ def save_playlist(self, pl, overwrite=False):
                 playlist should it happen to already exist
         """
         name = pl.name
+
+        # Make sure name is unicode
```

Comment:
Is it possible for `pl.name` to be an str? If it's always unicode we can just `assert(isinstance(name, unicode))` here, which will help us clarify the types in the future when moving to Python 3.

Reply:
At the moment, it does seem to happen that ```pl.name``` is a str object:
1) Open a new playlist
2) Rename a playlist
3) Close exaile - the conversion code path will be triggered

Although on subsequent open & close, this does not seem to be the case anymore, so it might also be a bug in the name-setting codepath?